_posts/2017-03-07-hddtools.md

```diff
@@ -0,0 +1,323 @@
+
+# Highight in red the stations on the Tanaro River
+plot(Italy, col = NA, border = "darkgrey")
+plot(hydro, add = TRUE, col = "blue", pch = 1)
+plot(TanaroSP, add = TRUE, col = "red", pch = 1)
+```
+
+![](/assets/blog-images/2013-03-07-hddtools/unnamed-chunk-8-1.png)
+
+If `columnName` refers to a numeric field, in the `columnValue` other terms of comparison can be specified. For example, it is straightforward to find out that the country with the most longstanding monitoring stations is Germany (with 11 out of 15 stations with more than 150 years of recordings).
+
+
+```r
+# Filter GRDC stations with more than 150 years of recordings
+GRDC150 <- catalogueGRDC(columnName = "d_yrs", columnValue = ">150")
+
+# Which country has the most longstanding monitoring stations?
+table(GRDC150$country_code)
+```
+
+```
+## 
+## DE FI LT RO SE 
+## 11  1  1  1  1
+```
+
+```r
+# Where is the oldest station?
+oldest <- GRDC150[which(GRDC150$d_yrs == max(GRDC150$d_yrs)), 
+                  c("grdc_no", "river", "station", "d_yrs")]
+oldest
+```
+
+```
+## # A tibble: 1 × 4
+##   grdc_no      river station d_yrs
+##     <chr>      <chr>   <chr> <dbl>
+## 1 6340120 ELBE RIVER DRESDEN   208
+```
+
+The oldest stastion is in Dresden on the Elbe River. Let's now find out whether monthly data is available. For this we need the GRDC identification number (this is stored in the column grdc_no of the catalogue) and the function `tsGRDC()`.  
+
+
+```r
+# Monthly data extraction
+DresdenStation <- tsGRDC(stationID = oldest$grdc_no, plotOption = TRUE)
+```
+
+![](/assets/blog-images/2013-03-07-hddtools/unnamed-chunk-10-1.png)<!-- -->
+
+The plot above, shows that the river flow has had huge oscillations over time, is this an effect of anthropogenic changes and/or climate change? This is well out of the scope of this blog post, I leave the reader to look at trends from the mean monthly value `DresdenStation$mddPerYear`.
+
+### NASA's Tropical Rainfall Measuring Mission
+The Tropical Rainfall Measuring Mission (TRMM) is a joint mission between NASA and the Japan Aerospace Exploration Agency (JAXA) that uses a research satellite to measure precipitation within the tropics in order to improve our understanding of climate and its variability.
+
+The TRMM satellite records global historical rainfall estimation in a gridded format since 1998 with a daily temporal resolution and a spatial resolution of 0.25 degrees (spatial extent goes from -50 to +50 degrees latitude). This information is openly available for educational purposes and downloadable from an FTP server. The hddtools provides a function, called TRMM(), to download and convert a selected portion of the TRMM dataset into a raster-brick that can be opened in any GIS software.
+
+Although Pompeii and surrounding areas are not in the tropics, we can download precipitation map for the past months (but remember, values become less reliable moving away from the tropics!).
```

Comment:
Agree, I added a reminder. 

I also deleted the 3B43_V7 form the comment. This is just the code that corresponds to mean monthly precipitations products.